src/api/app/policies/attrib_policy.rb

```diff
@@ -39,4 +20,32 @@ def update?
   def destroy?
     create?
   end
+
+  private
+
+  # Parses AttribTypeModifiableBy and AttribNamespaceModifiableBy association
+  # collections and checks the users permissions.
+  #
+  # Permission check is based on:
+  #   - User
+  #   - Group association of user
+  #   - Role association of user (Only for AttribTypeModifiableBy)
+  def has_permissions_for_modifiables?(attrib_modifiable_bies)
```

Comment:
Apparently that's how we / rails calls the association. See https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/attrib_type.rb#L17

We already had a laugh in the mobprogramming session 😆